util.grace

```diff
@@ -396,31 +396,38 @@ method execDir {
     execDirCache
 }
 
-method file(name) on(origin) orPath(pathString) otherwise(action) {
+method files(names) on(origin) orPath(pathString) otherwise(action) {
     def locations = filePath.split(pathString)
     locations.addFirst(origin)
     if (locations.contains(outDir).not) then { locations.addFirst(outDir) }
         // it's important that outDir is first; if not, the compiler
         // might compile an imported file, but then be unable to find the
         // code that it just generated.
     if (locations.contains(execDir).not) then { locations.addLast(execDir) }
-    def candidate = name.copy
-    def originalDir = name.directory
-    if ((originalDir.size > 0) && {originalDir.first == "/"}) then {
-        if (candidate.exists) then {
-            return candidate
-        } else {
-            return action.apply ""
+    def candidates = sequence << names.map{n -> n.copy}
+    def originalDirs = sequence << names.map{n -> n.directory}
+    // This assumes that either everything in "names" is an absolute path, or nothing is
```

Comment:
I think that this comment should apply to the `if` on the next line, not to `originalDirs`.  But why is it a reasonable assumption?